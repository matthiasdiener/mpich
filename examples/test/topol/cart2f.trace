Note: Line numbers for system and library calls may be incorrect 
Begin traceback...
Called from [func: _etext], at 0xef706c4c, args=0xb 0x3 0xef7d1a84 0xef802888
Called from [func: mpi_dims_create_:F(0,11)], at 0x28a0, args=0x0 0x2 0x0 0x629a0
Called from [func: _MAIN_], at 0x2354, args=0x5d374 0x504d0 0x5d390 0x0
Called from [func: _etext], at 0xef77cc38, args=0x0 0x1 0x1 0x2e000000
Called from [func: start], at 0x2064, args=0x0 0x10 0xeffff488 0x50400
End traceback...
Note: Line numbers for system and library calls may be incorrect 
Begin traceback...
Called from [func: _etext], at 0xef706c4c, args=0xb 0x3 0xef7d1a84 0xef802888
Called from [func: mpi_dims_create_:F(0,11)], at 0x28a0, args=0x0 0x2 0x0 0x629a0
Called from [func: _MAIN_], at 0x2354, args=0x5d374 0x504d0 0x5d390 0x0
Called from [func: _etext], at 0xef77cc38, args=0x0 0x1 0x1 0x2e000000
Called from [func: start], at 0x2064, args=0x0 0x10 0xeffff488 0x50400
End traceback...
Note: Line numbers for system and library calls may be incorrect 
Begin traceback...
Called from [func: _etext], at 0xf76b2c4c, args=0xb 0x3 0xf7795a84 0xf7802888
Called from [func: mpi_dims_create_:F(0,11)], at 0x28a0, args=0x0 0x2 0x0 0x629a0
Called from [func: _MAIN_], at 0x2354, args=0x5d374 0x504d0 0x5d390 0x0
Called from [func: _etext], at 0xf7740c38, args=0x0 0x1 0x1 0x2e000000
Called from [func: start], at 0x2064, args=0x0 0x10 0xf7fff3f8 0x50400
End traceback...
Note: Line numbers for system and library calls may be incorrect 
Begin traceback...
Called from [func: _etext], at 0xef706c4c, args=0xb 0x3 0xef7d1a84 0xef802888
Called from [func: mpi_dims_create_:F(0,11)], at 0x28a0, args=0x0 0x2 0x0 0x629a0
Called from [func: _MAIN_], at 0x2354, args=0x5d374 0x504d0 0x5d390 0x0
Called from [func: _etext], at 0xef77cc38, args=0x0 0x1 0x1 0x2e000000
Called from [func: start], at 0x2064, args=0x0 0x10 0xeffff490 0x50400
End traceback...
Note: Line numbers for system and library calls may be incorrect 
Begin traceback...
Called from [func: _etext], at 0xef706c4c, args=0xb 0x3 0xef7d1a84 0xef802888
Called from [func: _mpi_dims_create_], at 0x2890, args=0x0 0x2 0x0 0x62358
Called from [func: _MAIN_], at 0x2354, args=0x5cdac 0x504a8 0x5cdc8 0x0
Called from [func: _etext], at 0xef77cc38, args=0x0 0x1 0x1 0x2e000000
Called from [func: start], at 0x2064, args=0x0 0x10 0xeffff488 0x50400
End traceback...
Note: Line numbers for system and library calls may be incorrect 
Begin traceback...
Called from [func: _etext], at 0xef706c4c, args=0xb 0x3 0xef7d1a84 0xef802888
Called from [func: _mpi_dims_create_], at 0x2890, args=0x0 0x2 0x0 0x62358
Called from [func: _MAIN_], at 0x2354, args=0x5cdac 0x504a8 0x5cdc8 0x0
Called from [func: _etext], at 0xef77cc38, args=0x0 0x1 0x1 0x2e000000
Called from [func: start], at 0x2064, args=0x0 0x10 0xeffff488 0x50400
End traceback...
