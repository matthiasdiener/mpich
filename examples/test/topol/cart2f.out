*** Testing cart2 from Fortran ***
# The maximum space allocated was 6096 bytes [0]
# The maximum space allocated was 6096 bytes [0]
# The maximum space allocated was 6096 bytes [0]
 Done with test
# The maximum space allocated was 5060 bytes [0]
*** Testing cart2 from Fortran ***
